### PR TITLE
Add bunch of GraphQL queries

### PR DESCRIPTION
**Pull request by davispuh**
Add bunch of GraphQL queries.

### CODE REVIEW

File: resources/GraphQL.js [hunk 1]
```diff
@@ -1,6 +1,24 @@
...
+let mutations = {};
+loadGraphQL(queries, "queries");
+loadGraphQL(mutations, "mutations");
 
 module.exports = Object.freeze({
-  get EVALUATE_CODE_QUERY() { return read(`${__dirname}/graphql/queries/evaluateCodeQuery.graphql`).toString(); },
-  get REDEEM_CODE_MUTATION() { return read(`${__dirname}/graphql/mutations/redeemCodeMutation.graphql`).toString(); },
+  queries: Object.freeze(queries),
+  mutations: Object.freeze(mutations),
+  get EVALUATE_CODE_QUERY() { return queries.evaluateCodeQuery; },
```
Comment (Revadike) [inline resources/GraphQL.js:25]: I assume this is kept to support older code?

File: resources/GraphQL.js [hunk 2]
```diff
@@ -1,6 +1,24 @@
...
+    if (e.isFile() && path.extname(e.name) == '.graphql') {
+      let name = path.basename(e.name, '.graphql');
+      target[name] = read(`${dir}/${e.name}`).toString();
+    }
+  });
+}
+
+let queries = {};
+let mutations = {};
+loadGraphQL(queries, "queries");
```
Comment (Revadike) [inline resources/GraphQL.js]: Since `queries` and `mutations` are already global variable, I'd not pass them as parameters to avoid confusion.

**Review by Revadike — COMMENTED** [review]
Thanks for the PR! 
Also, I have a bunch more graphql queries/mutations collected, which can be added.
I don't have them on this current PC unfortunately. I'll share them later.

File: resources/GraphQL.js (hunk 1 above)
Comment (davispuh) [reply resources/GraphQL.js:25]: Yeah, I didn't want to break existing code if anyone relies on using these.
But if we bump major version we can remove them.

File: resources/GraphQL.js (hunk 2 above)
Comment (davispuh) [reply resources/GraphQL.js]: How do you imagine this implementation then? Currently `loadGraphQL` doesn't rely on any global variables. You can pass any object to get filled with GraphQL documents.

I don't see what confusion can happen here.
But actually we can just simply return them.

So
```js
let queries = getGraphQL("queries");
let mutations = getGraphQL("mutations");
```

**Comment by davispuh** [comment]
> Also, I have a bunch more graphql queries/mutations collected, which can be added.
> I don't have them on this current PC unfortunately. I'll share them later.

Yeah that would be nice, just name them same as I did based on method name and use https://beautifier.io/ to format same style (or use local/IDE formatter).

File: resources/GraphQL.js (hunk 1 above)
Comment (Revadike) [reply resources/GraphQL.js:25]: Yeah, I agree.

File: resources/GraphQL.js (hunk 2 above)
Comment (Revadike) [reply resources/GraphQL.js]: Yeah, that's better.

**Comment by davispuh** [comment]
@Revadike updated PR

**Review by Revadike — APPROVED** [review]

**Comment by Revadike** [comment]
@davispuh seems the graphql queries are no longer working or at least some of them.

**Pull request merged**